experiment/theory.md

```diff
@@ -1 +1,46 @@
-### Link your theory in here
\ No newline at end of file
+When two atoms combine to form a stable covalent moleculer, there are two repulsion forces acting between the two hetero atoms. One between the positively charged nuclei of both the atoms and the other between the negative electron clouds. The other force is the attraction between the nucleus of one atom with the electons of the other atom. Balancing the forces between them, the two atoms settle at a mean internuclear distance or the bond length where the total energy of the system is minimum. Any change like pulling the atoms away or squeezing them brings change in the bond length which requires an input of energy.
```

Comment:
The Theory has multiple grammatical mistakes, it needs to be reviewed by an SME

Reply:
SME still working on the Theory part. Sent the questions for now. Will be adding those questions to the pretest and the postest.